pkg/services/meta/meta.go

```diff
@@ -61,6 +61,48 @@ type wsClient interface {
 	Close()
 }
 
+func newNotifier() objectNotifier {
+	return objectNotifier{
+		notifications: make(chan oid.Address, 1024),
+		subs:          make(map[oid.Address]chan<- error),
+	}
+}
+
+type objectNotifier struct {
+	notifications chan oid.Address
+
+	m    sync.Mutex
+	subs map[oid.Address]chan<- error
+}
+
+func (on *objectNotifier) subscribe(addr oid.Address, ch chan<- error) {
+	on.m.Lock()
+	defer on.m.Unlock()
+
+	on.subs[addr] = ch
+}
+
+func (on *objectNotifier) unsubscribe(addr oid.Address) {
+	on.m.Lock()
+	defer on.m.Unlock()
+
+	delete(on.subs, addr)
+}
+
+func (on *objectNotifier) notifyRecieved(oid oid.Address) {
```

Comment:
`addr` is closer and avoids pkg name collision

Reply:
yeah, changed to addr in the last moment and have not fixed every place